### ISSUE TITLE
Delete group triggers disband group behaviour

### ISSUE THREAD

**Issue by lgrahl** [threema-ch/threema-web#556]
## Expected Behavior

If I **delete** the group, it should be deleted and the conversation should be removed as well.

## Current Behavior

The group is being disbanded (everyone is being kicked from the group, including me) but the conversation remains.

**Comment by dbrgn** [comment]
Definitely! That's a long-standing issue (the current behavior mirrored the delete button in the Android app), now that the behavior has been improved in the app we should provide the same buttons:

- Empty chat
- Leave group
- Delete group

When deleting a group, appropriate warnings should be shown (they might be different depending on whether you're the admin or not).

**Comment by mimiks** [comment]
Note that the confirmation dialogs for "leave" and "delete" are different in the app depending on if you're a group administrator or not.
